src/InstrumentationEngine/CorHeaders.h

```diff
@@ -178,10 +178,10 @@ typedef struct tagCOR_ILMETHOD_TINY : IMAGE_COR_ILMETHOD_TINY
 {
     bool     IsTiny () const         { return (Flags_CodeSize & (CorILMethod_FormatMask >> 1)) == CorILMethod_TinyFormat; }
     unsigned GetCodeSize () const    { return ((unsigned) Flags_CodeSize) >> (CorILMethod_FormatShift-1); }
-    unsigned GetMaxStack () const    { return 8; }
+    static unsigned GetMaxStack ()   { return 8; }
```

Comment:
>static unsigned GetMaxStack ()   { return 8; } [](start = 4, length = 46)

I would suggest just disabling the "function could be static" warning. There are often reasons that functions are not static when they could be.